lib/testium-core.js

```diff
@@ -11,12 +11,15 @@ var resolveDriver = require('./driver-factory');
 var initTestiumOnce = _.once(initTestium);
 var getConfig = _.once(Config.load);
 
+var DEFAULT_PAGE_SIZE = { height: 768, width: 1024 };
+
 // For backwards-compatibility
 module.exports = initTestium;
 
 function clearCookies(testium) {
   return Bluebird
     .try(_.bindKey(testium.browser, 'clearCookies'))
+    .then(_.partial(debug, 'Cookies cleared'))
```

Comment:
Just a random consistency fix (e.g. the two others are instrumented with debug messages, so it seemed weird to leave this silent).